### PR TITLE
Change conversation.updated-at timestamp when new message is created

### LINKED ISSUE #113: A conversation never moves up to the top of the list when someone replies.
See the second half of the discussion here https://github.com/zooniverse/Panoptes-Front-End/issues/3627

### PR DESCRIPTION
**Pull request by simoneduca**
Attempts to fix #113 
Test `spec/services/message_service_spec.rb` fails, probably because I'm missing something.

### CODE REVIEW

File: spec/services/message_service_spec.rb [hunk 1]
```diff
@@ -39,6 +41,9 @@
       before(:each){ service.create }
       subject{ service.resource }
       its(:user){ is_expected.to eql current_user }
+      it 'should change the conversations updated_at timestamp' do
+        expect { service.build }.to change { conversation.updated_at }
```
Comment (marten) [inline spec/services/message_service_spec.rb]: My fault, this should be `conversation.reload.updated_at`

File: spec/services/message_service_spec.rb [hunk 2]
```diff
@@ -16,6 +16,8 @@
     it_behaves_like 'a service creating', Message do
       let(:recipients){ conversation.users - [current_user] }
 
+
+
```
Comment (marten) [inline spec/services/message_service_spec.rb]: Do we need these two lines?

Comment (simoneduca) [inline spec/services/message_service_spec.rb]: Nope sorry.

File: spec/services/message_service_spec.rb (hunk 1 above)
Comment (simoneduca) [inline spec/services/message_service_spec.rb]: After adding `reload`, it still fails
screen shot 2017-05-18 at 11 27 07

**Comment by simoneduca** [comment]
I still can't get the test to pass here https://github.com/zooniverse/Talk-Api/pull/117/files/fd8dba3c2c8d81edf5d1a61f124d158355c78134#r117213316
Can someone point me in the direction please?

**Pull request merged**